src/prefect/engine/runner.py

```diff
@@ -101,6 +101,11 @@ def __init__(self, state_handlers: Iterable[Callable] = None):
         self.state_handlers = state_handlers or []
         self.logger = logging.get_logger(type(self).__name__)
 
+    def __repr__(self) -> str:
+        return "<{}: {}>".format(
+            type(self).__name__, type(self.state_handlers).__name__
```

Comment:
I believe `type(self.state_handlers).__name__` will always evaluate to the word `list`.

Reply:
Ah, I wasn't aware if any other iterable would be in use. I'll change it to list.

Reply:
For this just `<"Runner">` would actually be OK.  This class is never actually used directly, only subclassed.

Reply:
I've changed this to remove the `type(self.state_handlers).__name__`, and it is now:
`return "<{}: iterable>".format(type(self).__name__)`. I felt the word utterable would be a better option than list, just in case a tuple or set is passed.

Reply:
Fair; however, I don't think the type of `self.state_handlers` is very informative here.  I think maybe just `<"Runner">` is the way to go as this class doesn't have any descriptive attributes in general.

Reply:
Alright. I'll change this.